test/tools/libtesteth/ImportTest.cpp

```diff
@@ -91,8 +93,9 @@ bytes ImportTest::executeTest()
 					continue;
 
 				eth::Network network = networks[j];
-				std::pair<eth::State, ImportTest::execOutput> out = executeTransaction(network, m_envInfo, m_statePre, m_transactions[i].transaction);
-				m_transactions[i].postState = out.first;
+				results out = executeTransaction(network, m_envInfo, m_statePre, m_transactions[i].transaction);
```

Comment:
Use `std::tie()` to assign `postState` and `changeLog` and ignore tuple element of index 1.